### PR TITLE
add basic vm instance batch mode support

### PR DESCRIPTION
**Pull request by seanlong**
This a PR to fix https://github.com/zstackorg/zstack-dashboard/issues/19.
Currently only batch create & delete operations are supported.

### CODE REVIEW

File: ts/vm.ts [hunk 1]
```diff
@@ -982,20 +1011,33 @@ module MVmInstance {
...
-            $scope.oVmInstanceGrid.refresh();
-          });
+          var count = info.vmCount ? info.vmCount : 1;
+          for (var i = 0; i < count; ++i) {
+            var vm = new VmInstance();
+            info.uuid = info.resourceUuid = Utils.uuid();
+            info.state = 'Starting';
+            angular.extend(vm, info);
+            vm = vmMgr.wrap(vm);
+            $scope.oVmInstanceGrid.add(vm);
```
Comment (shawngao5) [inline ts/vm.ts]: rename vm as xxx-1 xxx-2. don't use same name for all vms.

File: ts/vm.ts [hunk 2]
```diff
@@ -982,20 +1011,33 @@ module MVmInstance {
...
+            vmMgr.create(info, (ret: VmInstance) => {
```
Comment (boogunote) [inline ts/vm.ts]: Please rename vm as xxx-1 xxx-2. Don't use same name for all vms.

File: zstack_dashboard/static/templates/vm/createVm.html [hunk 3]
```diff
@@ -2,7 +2,16 @@
     <div class="row">
         <div class="col-sm-15">
             <div class="tab-content">
-                <div id="createVmInstanceInfo" class="tab-pane active">
+                <div ng-if="isBatchMode()" id="createVmInstanceCount" class="tab-pane active">
```
Comment (boogunote) [inline zstack_dashboard/static/templates/vm/createVm.html]: This small page will let following pages inherent the same offset to top and part of page will out of view. Maybe move count input control to next page is better.

**Comment by seanlong** [comment]
changed as suggested.

**Comment by boogunote** [comment]
lgtm

**Comment by youyk** [comment]
Good work. Merged !

**Pull request merged**